### PR TITLE
fix: add nawed2611

### LINKED ISSUE #1224: Nawed Ali
## Profile Screenshot

image

### PR DESCRIPTION
**Pull request by nawed2611**
resolves #1224 

## Checklist for profile addition

- [ x ] Name the PR `fix: add <your-username>`.
- [ x ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ x ] You've linked the PR to the correct issue.

### CODE REVIEW

**Review by kabir0x23 — APPROVED** [review]
The code looks right ✔

**Review by Pradumnasaraf — APPROVED** [review]

**Comment by Pradumnasaraf** [comment]
Hey! @schmelto can you have a look at why the Prettier workflow is failing. Syntax seems okay!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: profiles/nawed2611.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/83456083/178050998-5d11d3df-e779-474e-8ea9-12e8fd782968.png",
+  "issueId": 1224,
+  "name": "Nawed Ali",
+  "username": "nawed2611"
+}
```
Comment (schmelto) [inline profiles/nawed2611.json:6]:
```suggestion
{
  "image": "https://user-images.githubusercontent.com/83456083/178050998-5d11d3df-e779-474e-8ea9-12e8fd782968.png",
  "issueId": 1224,
  "name": "Nawed Ali",
  "username": "nawed2611"
}

```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: profiles/nawed2611.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "image": "https://user-images.githubusercontent.com/83456083/178050998-5d11d3df-e779-474e-8ea9-12e8fd782968.png",
+  "issueId": 1224,
+  "name": "Nawed Ali",
+  "username": "nawed2611"
+}
+
```
Comment (schmelto) [inline profiles/nawed2611.json:7]:
```suggestion
```

**Comment by schmelto** [comment]
> Hey! @schmelto can you have a look at why the Prettier workflow is failing. The syntax seems okay!

No clue why prettier fails here. I've ran `npm run format` on the file and it seems to be that there is a change but in the working tree I cannot spot a difference.

image

Strange thing. I merge the changes and will run prettier on the whole project.

**Comment by schmelto** [comment]
Ok now prettier says its fine but I'm unsure what was changed in the file -> https://github.com/EddieHubCommunity/awesome-github-profiles/commit/49983cdbae546e3910e1fb0122121a9c6912be67

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Pull request merged**